backend/resolver/resolver.go

```diff
@@ -16,7 +16,10 @@ import (
 	resolverv1 "github.com/lyft/clutch/backend/api/resolver/v1"
 )
 
-const OptionAll = "__ALL__"
+const (
+	OptionAll                = "__ALL__"
+	DefaultAutocompleteLimit = 50
```

Comment:
should we have this be config driven?

Reply:
Yeah sure we can do that so a user can override the default.

Reply:
Actually I'm going to hold off on this for now, we dont have any configuration currently in place for the resolver and slotting it in does not seem straight forward at the moment. We can tackle this in a separate PR as its going to overload this one. If it was simply i would of just added it now.